packages/terra-button/src/Button.jsx

```diff
@@ -46,6 +46,10 @@ const propTypes = {
    */
   text: PropTypes.string,
   /**
+   * Sets the button type. One of button, submit, or reset
+   */
+  type: PropTypes.oneOf(['button', 'submit', 'reset']),
```

Comment:
Did we come to a final consensus on using  enum objects for known prop values or strings?